src/main/resources/templates/c_plus_plus/exercise/.clang-format

```diff
@@ -0,0 +1,8 @@
+---
+Language: Cpp
+BasedOnStyle:  Google
+IndentWidth: 4
+ColumnLimit: 120
+PointerAlignment: Left
+DerivePointerAlignment: false
+IncludeBlocks: Preserve
```

Comment:
These settings are a rather opinionated preference of mine, I think it might make sense to just rely on one of the defaults instead

Reply:
I changed it to only have the Google base style. But I kept the `IncludeBlocks: Preserve`. The Google base style can't handle `.hpp` includes.